sos/report/plugins/crio.py

```diff
@@ -50,6 +50,12 @@ def setup(self):
             "crio config"
         ])
 
+        result = self.exec_cmd("kill -USR1 $(pidof crio)")
+        if result['status'] == 0:
+            self.add_copy_spec([
+                "/tmp/crio-goroutine-stacks*.log"
+            ])
```

Comment:
`add_copy_spec()` supports single strings, so no need for a list here. Glob expansion is done later on.